crates/holochain_integrity_types/src/entry/app_entry_bytes.rs

```diff
@@ -59,6 +59,35 @@ impl From<AppEntryBytes> for SerializedBytes {
     }
 }
 
+/// Helpful pattern for debug formatting many bytes.
+/// If the size is > 32 bytes, only the first 8 and last 8 bytes will be displayed.
+pub fn many_bytes_string(bytes: &[u8]) -> String {
```

Comment:
Little bit bike-sheddy, but it is often also useful to see the size of the data when debugging, and hex is a little more compact. Might I suggest a format like `Binary { len: 42, bytes: "2a4204c2..dd3c100f" }`.

Reply:
@neonphog I've got a whole 'nother PR for this https://github.com/holochain/holochain/pull/2384